soc/nxp/rw/Kconfig

```diff
@@ -0,0 +1,81 @@
+# Copyright 2022-2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_RW6XX
+	select ARM
+	select CPU_CORTEX_M33
+	select CPU_CORTEX_M_HAS_DWT
+	select CLOCK_CONTROL
+	select PLATFORM_SPECIFIC_INIT
+	select CPU_HAS_ARM_SAU
+	select CPU_HAS_ARM_MPU
+	select CPU_HAS_FPU
+	select HAS_MCUX_OS_TIMER
+	select ARM_TRUSTZONE_M
+	select CPU_CORTEX_M_HAS_SYSTICK
+	select HAS_MCUX
+	select HAS_MCUX_FLEXCOMM
+	select INIT_SYS_PLL
+	select HAS_MCUX_CACHE
+	select HAS_PM
+
+if SOC_SERIES_RW6XX
+
+config INIT_SYS_PLL
+	bool "Initialize SYS PLL"
+
+menuconfig NXP_RW6XX_BOOT_HEADER
+	bool "Create boot header"
+	default y
+	help
+	  Create data structures required by the boot ROM to boot the
+	  application from an external flash device.
+
+if NXP_RW6XX_BOOT_HEADER
+
+choice BOOT_DEVICE
```

Comment:
We can probably drop the `BOOT_DEVICE` choice, right? I think we handle this with `zephyr,flash` node now